zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/security/Pkcs1SupportTest.java

```diff
@@ -62,6 +66,10 @@ void shouldSupportPkcs1Key(final @TempDir Path tmpDir) throws IOException, Certi
       zeebe.start();
 
       // then
+      SslAssert.assertThat(
+              new InetSocketAddress(
+                  zeebe.getExternalHost(), zeebe.getMappedPort(8080)))
```

Comment:
🔧 The formatter suggests this as a one-liner.
```suggestion
              new·InetSocketAddress(zeebe.getExternalHost(),·zeebe.getMappedPort(8080)))
```